microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -19,11 +19,11 @@ WebView2 is available for the following frameworks (or platforms) and programmin
    * [WPF](/dotnet/api/microsoft.web.webview2.wpf)
    * [Windows Forms](/dotnet/api/microsoft.web.webview2.winforms)
 
-* WinRT for WinUI 2 (UWP)
-   * [Core/C#](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
-   * [COM Interop/C++](/microsoft-edge/webview2/reference/winrt/interop/index)
+* WinRT for WinUI 2 (UWP) 
```

Comment:
Instead of this hierarchy I would recommend something as follows. Core and interop are common to any consumer of WinRT. WinUI2/3 are UI frameworks with their own specific WebView2 classes that should be linked.

 * WinRT
     * Core
     * COM Interop/C++
     * WinUI 2
         * Microsoft.UI.Xaml.Controls.WebView2 Class (or whatever it is - this link seems to be missing from the doc currently)
     * WinUI 3
          * Microsoft.UI.Xaml.Controls.WebView2 Class